### PR TITLE
Enrich input updates and reporting enhancement

### PR DESCRIPTION
**Pull request by bagrat**
* Update enrichment input to have `mcc`
* Add new account holder types - `freelance` and `unknown`
* Make reporting detailed, allowing to specify expected values

### CODE REVIEW

File: ntropy_sdk/ntropy_sdk.py [hunk 1]
```diff
@@ -160,9 +164,28 @@ def __init__(
     def __repr__(self):
         return f"EnrichedTransaction(transaction_id={self.transaction_id}, merchant={self.merchant}, logo={self.logo}, labels={self.labels})"
 
-    def report(self):
+    def report(
+        self,
+        **kwargs,
+    ):
+        supported_fileds = [
```
Comment (wuurrd) [inline ntropy_sdk/ntropy_sdk.py:171]: supported_fields

Comment (bagrat) [reply ntropy_sdk/ntropy_sdk.py:171]:
```suggestion
        supported_fields = [
```

**Review by wuurrd — APPROVED** [review]

File: tests/test_sdk.py [hunk 2]
```diff
@@ -1,12 +1,68 @@
...
+    for t in ACCOUNT_HOLDER_TYPES + ["not_valid"]:
+        if t in ACCOUNT_HOLDER_TYPES:
+            tx = create_tx(t)
+            assert tx.account_holder_type == t
+        else:
+            with pytest.raises(ValueError):
+                create_tx(t)
+
+
+def test_fileds():
```
Comment (wuurrd) [inline tests/test_sdk.py:40]: test_fields

Comment (bagrat) [reply tests/test_sdk.py:40]:
```suggestion
def test_fields():
```

**Pull request merged**